packages/discovery-provider/src/app.py

```diff
@@ -425,6 +426,10 @@ def configure_celery(celery, test_config=None):
                 "task": "create_engagement_notifications",
                 "schedule": timedelta(minutes=10),
             },
+            "create_listen_streak_reminder_notifications": {
+                "task": "create_listen_streak_reminder_notifications",
+                "schedule": timedelta(seconds=10),
```

Comment:
i set it to 10 seconds for dev but i think it could be every hour on prod probably